JeffShepherd/GPSammich
# Feature/cards

## Is this a fix or a feature? ⬇️

feature

## What is the change? ⬇️

Update card content, positioning, and styles

## How should this be tested? ⬇️

Open index.html in a browser and ensure that the layout roughly follows the comp provided in the project specs. Make sure each card appears to be spaced evenly and looks good.
